net/prometheus/Portfile

```diff
@@ -42,14 +42,18 @@ set prom_share_dir  ${prefix}/share/${name}
 set prom_log_dir    ${prefix}/var/log/${name}
 set prom_log_file   ${prom_log_dir}/${name}.log
 
-checksums   rmd160  f631fa9b637a30186902d838fe0d3c5e9e2f14bd \
-            sha256  b5e508f1c747aaf50dd90a48e5e2a3117fec2e9702d0b1c7f97408b87a073009 \
-            size    12820441
+checksums   rmd160  f18d4e10b2f34ee2721e7cdee5c83f46ad52511e \
+            sha256  d0b53411ea0295c608634ca7ef1d43fa0f5559e7ad50705bf4d64d052e33ddaf \
+            size    12822668
 
 add_users           ${prom_user} \
                     group=${prom_user} \
                     realname=Prometheus
 
+pre-build {
+  system "env"
```

Comment:
Hi @herbygillot, out of curiosity, what's the purpose of this `pre-build` section?

Reply:
Hey @emcrisostomo - I had mentioned this in the description, but `prometheus` has been failing to build in the MacPorts buildbot environment for a while now, related to a call in Go to get the current user.  The latest error messages we're seeing seem to suggest that perhaps it could be because the USER environment variable isn't being set.  This would help give us an idea.

This issue is being tracked here: https://trac.macports.org/ticket/58709